gentoo: ld: cannot find now: No such file or directory

Unable to build gentoo rootfs image

```
-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2480: git-credential-store] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:2480: git-daemon] Error 1
make: *** [Makefile:2480: git-fast-import] Error 1
make: *** [Makefile:2480: git-http-backend] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: make: *** [Makefile:2480: git-sh-i18n--envsubst] Error 1
warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2483: git-imap-send] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2494: git-remote-testsvn] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2480: git-shell] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2487: git-http-fetch] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2490: git-http-push] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2480: git-credential-cache--daemon] Error 1
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: -z -O1 ignored
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find now: No such file or directory
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find relro: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2480: git-credential-cache] Error 1
 * ERROR: dev-vcs/git-2.26.2::gentoo failed (compile phase):
 *   emake failed
```

--- a/rootfs-builder/gentoo/rootfs_lib.sh
+++ b/rootfs-builder/gentoo/rootfs_lib.sh
@@ -117,10 +117,9 @@ build_rootfs() {
 		-Wl,--copy-dt-needed-entries
 		-Wp,-D_REENTRANT
 		-Wl,--enable-new-dtags
-		-Wl,now
-		-Wl,relro
 		-Wl,-sort-common
-		-Wl,-z
+		-Wl,-z -Wl,now
+		-Wl,-z -Wl,relro
 	)
 
 	local build_dependencies=(
@@ -157,7 +156,7 @@ build_rootfs() {
 	quickpkg --include-unmodified-config=y "*/*"
 
 	# Install needed packages excluding conflicting packages
-	ROOT=${ROOTFS_DIR} emerge --exclude $(echo "${conflicting_packages[*]}") --newuse -k ${PACKAGES}
+	ROOT=${ROOTFS_DIR} emerge --exclude "$(echo "${conflicting_packages[*]}")" --newuse -k ${PACKAGES}
 
 	pushd ${ROOTFS_DIR}
 

